src/components/NotificationsList.js

```diff
@@ -70,16 +76,50 @@ class NotificationsList extends Component {
   };
 
   onOptionsPress = selectedNotification => {
-    this.setState({
-      selectedNotification,
-      optionsVisible: true
-    });
+    this.setState({ selectedNotification, optionsVisible: true });
+  };
+
+  onEmploymentConfirmPress = () => {
+    const { sentBy, employeeId } = this.state.selectedNotification;
+
+    if (this.state.isAcceptingEmployment) {
+      // Acepta la invitación
+      this.props.onEmployeeCreate({ commerceId: sentBy, employeeId, profileId: this.props.clientId });
+      this.props.onEmploymentInvitationConfirm(this.state.selectedNotification, true);
+      this.props.onUserWorkplacesRead();
+    } else {
+      // Rechaza la invitación
+      this.props.onEmployeeDelete({ commerceId: sentBy, employeeId, profileId: this.props.clientId });
+      this.props.onEmploymentInvitationConfirm(this.state.selectedNotification, false);
+      this.props.onEmploymentInvitationConfirm;
+    }
+
+    this.setState({ confirmEmploymentVisible: false });
+  };
+
+  onEmploymentInvitationPress = action => {
+    this.setState({ confirmEmploymentVisible: true, optionsVisible: false, isAcceptingEmployment: action });
+  };
+
+  renderEmploymentInvitationOptions = () => {
+    return (
+      <View>
+        <MenuItem
+          title="Aceptar Invitación"
+          icon="md-checkmark"
+          onPress={() => this.onEmploymentInvitationPress(true)}
+        />
+        <Divider style={{ backgroundColor: 'grey' }} />
+        <MenuItem title="Rechazar Invitación" icon="md-close" onPress={() => this.onEmploymentInvitationPress(false)} />
+        <Divider style={{ backgroundColor: 'grey' }} />
+      </View>
+    );
   };
 
   renderRow = ({ item }) => {
     return (
       <ListItem
-        title={item.title}
+        title={`${item.title}${item.acceptanceDate ? ' (Aceptada)' : item.rejectionDate ? ' (Rechazada)' : ''}`}
```

Comment:
en el invitationCancel le haces softDelete a la notificacion, asi que creeria que las rechazadas nunca se mostrarian o no?

Reply:
El invitationCancel se usa solo del lado del negocio y es para que se borre la notificacion que hace el negocio cuando invita a una persona a laburar... Para mi esa era la mejor forma, capaz invitó a alguien que no quiso entonces se debería borrar. Sino se la puede updatear como rechazada en vez de borrarla... La persona nunca va a cancelar una invitacion, sino rechazarla o aceptarla. Lo que me estarias diciendo al ultimo seria enviar notificacion al rechazar la invitacion, que tambien se puede agregar si quieren. Para mi que ya es demasiada notificación pero bueno como queráis

Reply:
Ah de una, flayie que habia visto el invitationCancel del lado del cliente, mala mia.
Y con respecto a lo de la notificacion, porque seria demasiada noitifcacion?? Tampoco que va a estar invitando todo el tiempo, y de paso esta bueno que se entere si acepto o no un empleado, sino como que tiene que ir a fijarse al listado.

Reply:
Bueno se la agrego a la notificacion esa